city/router.py

```diff
@@ -33,6 +32,6 @@ def delete_city(
     del_city = crud.delete_city(db=db, city_id=city_id)
 
     if del_city is None:
-        raise HTTPException(status_code=404, detail="City does not exist")
+        raise Response(status_code=status.HTTP_204_NO_CONTENT)
 
     return HTTPException(status_code=204, detail="City deleted")
```

Comment:
when del_city is None it means that city was not found(because you returned 404 error in the last implement) so why is it OK now? In the current implemented regardless of what is del_city, you always return success deletion 204 status code that message that city was delete. You fixed wrong response that Serhii mentioned